packages/odyssey-react/src/components/TextInput/TextInput.module.scss

```diff
@@ -34,11 +33,11 @@
   line-height: var(--LineHeight);
 
   &:hover,
-  &:focus {
+  &:focus-within {
```

Comment:
`.root` becomes the wrapper-parent, so we'll need `:focus-within` to detect when the internal input is focused.